scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -1740,18 +1909,18 @@
               offers [offer-6 offer-7]]
           (is (run-handle-resource-offers! num-considerable offers :gpu))
           (is (= :end-marker (async/<!! offers-chan)))
```

Comment:
Why do we not test mesos successfully launches here? Why is mesos not launching even though the test hasn't changed code?

Reply:
mesos doesn't support users requesting gpus via the `gpus` field. so these tests were previously written assuming that users could do so, but those actions are not in practice. we modified the constraint so mesos jobs that requested a positive number of gpus would not successfully submit (only mesos jobs that request 0 gpus can submit)